pkgs/games/keeperrl/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv, fetchFromGitHub, makeWrapper, requireFile
+, openal, curl, libogg, libvorbis
+, SDL2, SDL2_image, zlib
+, unfree_assets ? false }:
+
+with stdenv.lib;
+
+let
+  pname = "keeperrl";
+
+in stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
```

Comment:
he references {name} in the rest of the derivation though

Reply:
I think I'm getting majorly confused by the nested rec sets but aren't the two uses of `name` from the inner rec set?

Reply:
oh, i think you're right, it would get shadowed. That is very confusing.

Reply:
I think that is a further reason to get rid of name here, and potentially `assets` should be defined in a let clause at the top because I think it is quite unclear atm

Reply:
I'll use "pname" and remove "name", but will leave "assets" where it is, since it makes use of "version."